include/class.api.php

```diff
@@ -174,7 +174,7 @@ function requireApiKey() {
         # header
 
         if(!($key=$this->getApiKey()))
-            return $this->exerr(401, __('Valid API key required'));
+            return $this->exerr(401, __('Valid API key required: ').$_SERVER['REMOTE_ADDR']);
```

Comment:
I think this was _only_ staged when I was debugging proxies \ load balancers setup... I don't think it's worth changing translation string for it, plus I think returning generic error makes sense from security standpoint. 

That said, we can perhaps log a detailed debug message to the system logs with the IP in the future for debug purposes.